src/staff_rename.lua

```diff
@@ -3,8 +3,8 @@ function plugindef()
     -- are both reserved for the plug-in definition.
     finaleplugin.Author = "Jacob Winkler"
     finaleplugin.Copyright = "2022"
-    finaleplugin.Version = "1.1"
-    finaleplugin.Date = "3/18/2022"

```

Comment:
Nit: it looks like this is for renaming selected staves? If so, does it require a selection? If so, please add the following:

```lua
finaleplugin.RequireSelection = true
```

https://www.finalelua.com/docs/rgp-lua/finaleplugin-properties#requireselection-boolean

Reply:
If you make a selection, it processes the selected staves. If you do not, it processes them all.